### ISSUE TITLE
X6 printer (TinyPrint) compatiblity 

### ISSUE THREAD

**Issue by KaupMa** [Mywk/iPrintUtility#1]
TinyPrint printer compatibility. X6 printers that use the TinyPrint application also work with the phone's iPrint application. But this iPrintUtility cannot find that printer. Would it be just device filtering that hides the printer? Windows 11 found and paired with the X6h-3412 printer (3412 is part of mac-address 12:34:56:78:9a:bc).
Could the search be done in such a way that you can also select an unknown device?

**Comment by WallK** [comment]
I'd like to have this feature too
My printer advertises as sc03h

**Comment by Mywk** [comment]
Theoretically I could make it so it shows all devices even if not supported, will look into this.

**Comment by decaffeinatedcoffee** [comment]
i have the same problem but with a walkPrint one

**Comment by xtcian** [comment]
mx10 support would be nice.. it doesnt detect it tho.. even tho its paired in windows

**Comment by Mywk** [comment]
Try with the latest version. While I added no functionality per se, assuming it uses the same communication method as the supported printers just click the "Force show all devices" and try to connect.

**Comment by SilverGreen93** [comment]
The "Force show all devices" is not intuitive at all. You can never tell if it is pressed or not. My suggestion would be to make it a checkbox.

The X6h is now detected. Paper feed and retract works fine. Printing usually works fine, but sometimes the last line is printed only in half (horizontally).

Found this python project that has support for X6h: https://github.com/Dejniel/TiMini-Print maybe it can help further.

All in all, I think you can add it as a supported printer.

**Comment by Mywk** [comment]
Yeah I agree the toggle button is not intuitive at all with this theme. Does the X6h show exactly like that or is it a partial name for the printer?

**Comment by SilverGreen93** [comment]
> Yeah I agree the toggle button is not intuitive at all with this theme. Does the X6h show exactly like that or is it a partial name for the printer?

It shows something like X6h-1234, as @KaupMa said in the first post.
